From 094cec37ef38356c02d0b8723f79271cdd6fc0d4 Mon Sep 17 00:00:00 2001
From: Gauthier Piarrette <gauthierpiarrette@gmail.com>
Date: Mon, 12 Oct 2026 14:39:28 +0200
Subject: [PATCH 1/6] vulkan_procs:
 vkGetPhysicalDeviceCooperativeMatrixPropertiesKHR is optional

winevulkan on MoltenVK returns NULL for it, and a missing instance entry point was fatal, so no device could be
created.
---
 libs/vkd3d/vulkan_procs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/vkd3d/vulkan_procs.h b/libs/vkd3d/vulkan_procs.h
index 3dbb37d..851b75e 100644
--- a/libs/vkd3d/vulkan_procs.h
+++ b/libs/vkd3d/vulkan_procs.h
@@ -385,7 +385,7 @@ VK_DEVICE_EXT_PFN(vkGetLatencyTimingsNV)
 VK_DEVICE_EXT_PFN(vkQueueNotifyOutOfBandNV)
 
 /* VK_KHR_cooperative_matrix */
-VK_INSTANCE_PFN(vkGetPhysicalDeviceCooperativeMatrixPropertiesKHR)
+VK_INSTANCE_EXT_PFN(vkGetPhysicalDeviceCooperativeMatrixPropertiesKHR)
 
 /* VK_EXT_opacity_micromap */
 VK_DEVICE_EXT_PFN(vkGetMicromapBuildSizesEXT)
-- 
2.54.0 (Apple Git-157)

